### DISCUSSION TITLE
Keep running debug when browser closed

### DISCUSSION THREAD

**Discussion by txwdzxq** [coder/code-server#6564] [Q&A] [answered]
code-server: v4.19.1
Code: 1.84.2

Running Python scripts on code server will continue to execute even if the browser is closed.

However, closing the browser during debugging and then reopening the code server will not maintain a breakpoint state.

Closing the browser while running a script in ipynb file (jupyter plugin) also interrupt program execution, which may not be the same reason as the debugging issue mentioned above.

Is there currently a way to keep debug or ipynb files running while closing or refreshing the browser?

**Comment by code-asher** [comment, marked as answer]
There is no way currently, not that I know of at least.  Here is the upstream issue: https://github.com/microsoft/vscode/issues/188413

**Reply by txwdzxq** [reply]
It seems that I still need to deploy a jupyter~
